libs/vkd3d/command.c

```diff
@@ -772,6 +772,9 @@ static HRESULT d3d12_fence_signal(struct d3d12_fence *fence, uint64_t physical_v
         return hresult_from_errno(rc);
     }
 
+    if (fence->physical_value > physical_value)
+        return S_OK;
```

Comment:
I guess this can theoretically happen if we submit Signal(1) -> Wait(1) -> Signal(2) on different queue threads, and we observe the signal 2 before signal 1?

Reply:
Probably a good idea to comment how this can happen.

Reply:
Done.